cupy/_logic/truth.py

```diff
@@ -98,6 +98,72 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     return v1 == v2 if invert else v1 != v2
 
 
+def intersect1d(arr1, arr2, assume_unique=False, return_indices=False):
+    """Find the intersection of two arrays.
+    Returns the sorted, unique values that are in both of the input arrays.
+
+    Parameters
+    ----------
```

Comment:
Please, keep the docstring format consistent with the code base

Reply:
Sorry, it seems that numpy docstrings style is actually ok. https://github.com/cupy/cupy/issues/5437#issuecomment-879148251
My bad! 😅

thanks to @grlee77 for pointing this out to me!

Reply:
No worries :v: 

I think it will be good to update other functions too. May I go for it?